react/features/mobile/navigation/components/overlay/components/LoadConfigOverlay.js

```diff
@@ -3,12 +3,11 @@
 import React, { PureComponent } from 'react';
 import { SafeAreaView, Text, View } from 'react-native';
 
-import { translate } from '../../../base/i18n';
-import { LoadingIndicator } from '../../../base/react';
-import { StyleType } from '../../../base/styles';
-
-import OverlayFrame from './OverlayFrame';
-import styles, { TEXT_COLOR } from './styles';
+import { translate } from '../../../../../base/i18n';
```

Comment:
Please move this to the navigation feature and turn it into a screen, drop overlay from the name.